flowetl/flowetl/flowetl/util.py

```diff
@@ -261,12 +271,6 @@ def create_dag(
     from flowetl.sensors.table_flux_sensor import TableFluxSensor
 
     # Choose flux sensor
-    if use_file_flux_sensor is not None:
-        message = "The 'use_file_flux_sensor' argument is deprecated."
-        if use_flux_sensor == True:
-            use_flux_sensor = "file" if use_file_flux_sensor else "table"
-            message = f"{message} Set use_flux_sensor='{use_flux_sensor}' instead."
-        warnings.warn(message, DeprecationWarning)
```

Comment:
If we're removing the deprecation warning, can we also remove the `use_file_flux_sensor` argument, please?

Reply:
:+1:

Reply:
Done and changelog entry added